### PR TITLE
Remove partner access token dependency

### PR DESCRIPTION
**Pull request by meetkoriya13**

### CODE REVIEW

File: src/lib/api/services/url.ts [hunk 1]
```diff
@@ -136,18 +136,21 @@ export const URLS = {
 
     // Extension
     REGISTER_EXTENSION: (): string => {
-        return urlJoin(MIXMASTER_URL('panel'), `/extensions/`);
+        return urlJoin(
+            MIXMASTER_URL('partner'),
+            `organization/${organization_id}/extension`
```
Comment (jinalvirani) [inline src/lib/api/services/url.ts:141]: why endpoint changed from /extensions to /extension ??

File: src/lib/api/services/url.ts [hunk 2]
```diff
@@ -136,18 +136,21 @@ export const URLS = {
...
+            MIXMASTER_URL('partner'),
+            `organization/${organization_id}/extension/${extension_api_key}`
+        )
     },
     UPDATE_EXTENSION_DETAILS: (extension_api_key: string): string => {
         return urlJoin(
-            MIXMASTER_URL('panel'),
-            `/extensions/${extension_api_key}`,
+            MIXMASTER_URL('partner'),
+            `organization/${organization_id}/extension/${extension_api_key}`,
```
Comment (jinalvirani) [inline src/lib/api/services/url.ts:153]: why endpoint changed from /extensions to /extension ??

File: src/lib/api/services/url.ts [hunk 3]
```diff
@@ -136,18 +136,21 @@ export const URLS = {
...
+            `organization/${organization_id}/extension`
+        )
     },
     GET_EXTENSION_DETAILS: (extension_api_key: string): string => {
         return urlJoin(
-            MIXMASTER_URL('panel'),
-            `/extensions/details/${extension_api_key}`,
-        );
+            MIXMASTER_URL('partner'),
+            `organization/${organization_id}/extension/${extension_api_key}`
```
Comment (jinalvirani) [inline src/lib/api/services/url.ts:147]: why endpoint changed from /extensions to /extension ??

File: src/lib/api/services/url.ts [hunk 4]
```diff
@@ -136,18 +136,21 @@ export const URLS = {
...
+        )
     },
     UPDATE_EXTENSION_DETAILS: (extension_api_key: string): string => {
```
Comment (jinalvirani) [inline src/lib/api/services/url.ts:150]: extension_api_key param name is misguiding to pass api key, where as we query this param agains _id in api. please update all occurrences of it.

File: src/lib/api/services/extension.service.ts [hunk 5]
```diff
@@ -14,12 +16,15 @@ type UpdateLaunchURLPayload = {
...
+        data: RegisterExtensionPayload,
     ) => {
         try {
             let headers = getCommonHeaderOptions().headers;
-            headers['x-partner-token'] = partner_access_token;
+            data.scope = ['company/profile'];
+            data.logo = { 
+                small: 'https://res.cloudinary.com/dwzm9bysq/image/upload/v1566539375/production/media/store/logo/jwosxsgh9ufoucdxpm10.png',
+                large: 'https://res.cloudinary.com/dwzm9bysq/image/upload/v1566539375/production/media/store/logo/jwosxsgh9ufoucdxpm10.png'
+            }
```
Comment (jinalvirani) [inline src/lib/api/services/extension.service.ts:30]: why not passing callbackes too?

Comment (jinalvirani) [reply src/lib/api/services/extension.service.ts:30]: developed_by_name and contact_email also

File: src/lib/api/services/url.ts [hunk 6]
```diff
@@ -136,18 +136,21 @@ export const URLS = {
...
         return urlJoin(
-            MIXMASTER_URL('panel'),
-            `/extensions/${extension_api_key}`,
+            MIXMASTER_URL('partner'),
+            `organization/${organization_id}/extension/${extension_api_key}`,
         );
     },
     GET_ORGANIZATION_DATA: (partner_access_token: string): string => {
```
Comment (jinalvirani) [inline src/lib/api/services/url.ts]: This is unused, please remove it

File: src/__tests__/previewUrlExtension.spec.ts [hunk 7]
```diff
@@ -112,7 +107,6 @@ describe('Extension preview-url command', () => {
 
         const promptSpy = jest
             .spyOn(inquirer, 'prompt')
-            .mockResolvedValueOnce({ partner_access_token: TOKEN })
```
Comment (jinalvirani) [inline src/__tests__/previewUrlExtension.spec.ts:115]: remove PARTNER_ACCESS_TOKEN from config.ts too

File: src/commands/partner.ts [hunk 8]
```diff
@@ -1,13 +0,0 @@
-import { Command } from 'commander';
-import Partner from '../lib/Partner';
-
-export default function partner(program: Command) {
-    const partner = new Command('partner').description('Partner Commands');
```
Comment (jinalvirani) [inline src/commands/partner.ts:5]: Remove this command from readme file

Comment (jinalvirani) [reply src/commands/partner.ts:5]: Aslo raise jira ticket to update partner documentation

Comment (jinalvirani) [reply src/commands/partner.ts:5]: Are we not handling proper deprecation message to user if they tries to hit this cmd?

File: src/lib/Partner.ts [hunk 9]
```diff
@@ -1,57 +0,0 @@
...
-            let spinner = new Spinner('Verifying Access Token');
-            try {
-                spinner.start();
-                organizationInfo = await ExtensionService.getOrganizationData(
-                    answers.partner_access_token,
-                );
-
-                if (!organizationInfo) {
-                    throw new CommandError(
-                        ErrorCodes.INVALID_PARTNER_TOKEN.message,
```
Comment (jinalvirani) [inline src/lib/Partner.ts:34]: INVALID_PARTNER_TOKEN remove this from commandErrors.ts

File: src/lib/api/services/url.ts (hunk 1 above)
Comment (meetkoriya13) [reply src/lib/api/services/url.ts:141]: changed panel server to partner server

File: src/lib/api/services/url.ts (hunk 2 above)
Comment (meetkoriya13) [reply src/lib/api/services/url.ts:153]: changed panel server to partner server

File: src/lib/api/services/url.ts (hunk 3 above)
Comment (meetkoriya13) [reply src/lib/api/services/url.ts:147]: changed panel server to partner server

File: src/lib/api/services/url.ts (hunk 4 above)
Comment (meetkoriya13) [reply src/lib/api/services/url.ts:150]: we are referring extension _id as extension_api_key in all the extension commands in fdk-cli

File: src/lib/api/services/extension.service.ts (hunk 5 above)
Comment (meetkoriya13) [reply src/lib/api/services/extension.service.ts:30]: 1. callbacks will be handled at api level
2. we are not taking that info from users when creating extension with cli

File: src/lib/api/services/url.ts (hunk 4 above)
Comment (pooransuthar1999) [reply src/lib/api/services/url.ts:150]: Same we are referring in the documentation as well thats why we are referring it as extension_api_key

File: src/lib/api/services/extension.service.ts (hunk 5 above)
Comment (pooransuthar1999) [reply src/lib/api/services/extension.service.ts:30]: As it is available on cli as well as  editable in future from developer side we are adding  from cli of whichever account they have logged in will be using that only

File: src/fdk.ts [hunk 10]
```diff
@@ -183,6 +177,15 @@ Run \`npm install -g ${packageJSON.name}\` to get the latest version.`;
                     }
                 }
             }
+            if(parent.args.includes('extension')) {
+                    const { version } = await ExtensionService.getFyndPlatformVersion();
+                    if(!semver.gte(version, '1.9.1')){
```
Comment (vivek-gofynd) [inline src/fdk.ts:182]: Define version as constant

File: src/lib/CommandError.ts [hunk 11]
```diff
@@ -72,7 +68,16 @@ export const ErrorCodes = {
         message: 'Network issue',
         code: 'FDK-0009',
     },
-
+    IN_COMPATIBLE_THEME_VERSION: {
+        message: (currentVersion)=> {
+         return `Your Fyndplatform version is currently ${currentVersion}. For compatibility, this fdk/cli requires Fyndplatform version 1.9.1 or newer. To align with your current Fynd Platform version, please downgrade your fdk/cli to any of the latest versions less than 5.0.0.`
```
Comment (vivek-gofynd) [inline src/lib/CommandError.ts:73]: make this error message dynamic based on FP version

**Pull request closed without merging**